libursa/Daemon.h

```diff
@@ -9,6 +9,9 @@
 #include "Responses.h"
 #include "Task.h"
 
+Response dispatch_command(const Command &cmd, Task *task,
+                          const DatabaseSnapshot *snap);
```

Comment:
I've made this routine public, since the new executables directly create command structures from the CLI arguments. Obvious alternative is to serialize the command into the textual format and pass it to `dispatch_command_safe` but this is extra work. Can do that if you don't want to expose `dispatch_command` directly.